ts/DataGrid/Utils.ts

```diff
@@ -124,6 +124,18 @@ namespace DataGridUtils {
 
         return 0;
     }
+
+    /**
+     * Check HTML tags in the string.
+     *
+     * @param str
+     * Text to verify.
+     *
+     * @returns
+     */
+    export function isHTML(str: string): boolean {
```

Comment:
`isHTML` or maybe `possiblyHTML`?

Reply:
Please no! 

In the core, we also use similar naming like `isString` etc.

Reply:
Only that `str.indexOf('<') === -1` means that it `isString` for 100%, and `str.indexOf('<') !== -1` means that there is a chance that it is HTML, but there is no such certainty yet. :D

But I can agree that it's OK if you specify it in the doclet and we're sure we won't be adding any util function in the future that more precisely checks if the text is 100% HTML.